src/Bridges/Constraint/bridges/quad_to_soc.jl

```diff
@@ -209,13 +215,25 @@ function MOI.delete(model::MOI.ModelLike, bridge::QuadtoSOCBridge)
     return
 end
 
-# Attributes, Bridge acting as a constraint
+function MOI.supports(
+    model::MOI.ModelLike,
+    attr::Union{MOI.ConstraintPrimalStart,MOI.ConstraintDualStart},
+    ::Type{QuadtoSOCBridge{T}},
+) where {T}
+    F, S = MOI.VectorAffineFunction{T}, MOI.RotatedSecondOrderCone
+    return MOI.supports(model, MOI.VariablePrimalStart(), MOI.VariableIndex) &&
+           MOI.supports(model, attr, MOI.ConstraintIndex{F,S})
+end
+
 function MOI.get(
     model::MOI.ModelLike,
-    attr::MOI.ConstraintPrimal,
+    attr::Union{MOI.ConstraintPrimal,MOI.ConstraintPrimalStart},
     bridge::QuadtoSOCBridge,
 )
     soc = MOI.get(model, attr, bridge.soc)
+    if soc == nothing
```

Comment:
```suggestion
    if soc === nothing
```
usually the convention for identity equality with nothing

Reply:
We can even use `isnothing` now that Julia v1.6 is the lower bound

Reply:
potentially yes, I never saw the interest of that function compared to equality check

Reply:
If it was better to do `=== nothing`, I guess the function wouldn't exist but IMO both are fine